shell/src/peer_manager.rs

```diff
@@ -91,17 +95,26 @@ impl From<std::io::Error> for NotifyProposerError {
     }
 }
 
+impl From<std::sync::mpsc::SendError<ProposerMsg>> for NotifyProposerError {
+    fn from(err: std::sync::mpsc::SendError<ProposerMsg>) -> Self {
+        Self::SendError(err)
+    }
+}
+
+pub type ProposerHandleRef = ProposerHandle;
```

Comment:
Why the alias?

Reply:
@binier 
peer_manager is not used anywhere, I just did not want to remove it to aviod rebase conflicts,
but when you approve this, I will clean it and merge to your branch

Reply:
@binier 
to that alias, I had there before:
```
pub type ProposerHandleRef = Arc<ProposerHandle>;
```
not needed, removing

Reply:
@binier 
removed all peer_manger riker actor related stuff + removed ProposerHandleRef